theano/tensor/nnet/opt.py

```diff
@@ -200,6 +200,7 @@ def local_abstractconv_gradinputs_gemm(node):
                              subsample=node.op.subsample,
                              filter_dilation=node.op.filter_dilation,
                              unshared=node.op.unshared)(kern, topgrad, shape)
+
```

Comment:
I don't know if these newlines are things that you have introduced yourself in a previous commit, but in general it's probably better not to have these changes that don't really do anything.